### ISSUE TITLE
gtf2bed example doesn't work

### ISSUE THREAD

**Issue by vreuter** [bedops/bedops#240]
Downloading the example file GTF from the latest docs page then running `gtf2bed < <filepath.gtf>` yields no output.
```
~/software/bedops_2.4.35/bin/gtf2bed < reference_gtf2bed_foo.gtf
```

**Comment by vreuter** [comment]
Issue I think is space delimiters not tabs.

**Comment by alexpreynolds** [comment]
Reopening so that I can investigate and fix this.

**Comment by vreuter** [comment]
One other minor thing I noticed was the link/filename mismatch. No big deal, but it was a momentary surprise to download `foo.gtf` but get `reference_gtf2bed_foo.gtf`. I guess makes sense to not have it named `foo.gtf` to protect against a user inadvertently overwriting an existing local file with that name upon download, but maybe the link text in the docs could change to match?

**Comment by alexpreynolds** [comment]
It's likely an issue with other conversion format example files. I'll try to take a look at all of this.

**Comment by vreuter** [comment]
Sounds good. I thought maybe I'd opened it in an editor that had toggled tabs to spaces but I redownloaded to double check.

**Comment by louisjrliu** [comment]
I'm having the same issue, is it resolved now?

**Comment by aupadhaya** [comment]
Hello there, I am having issue using convert2bed. I am using latest version. convert2bed worked with gff3 format but not gtf. GTF and gff3 files are from ensembl.

convert2bed -i GTF <annotation.gtf> annotation.bed

Error: Potentially missing gene or transcript ID from GTF attributes (malformed GTF at line [1]?)

**Comment by alexpreynolds** [comment]
Fixed in commit b51b52a1
